README.md

```diff
@@ -90,6 +90,10 @@ key.
   **watchdog later. Too low value may result in a reset loop!**
 * `j SD 3.0 mode` - enable SD controller in 3.0 mode to allow achieving full
   speeds with UHS-I SD cards
+* `g Reverse order of PCI addresses` - If disabled devices in mPCIe slots have
```

Comment:
I would write something like this:
```
When enabled, the PCIe devices are ordered as follows: NICs, mPCIe1, mPCIe2. In such way the PCI enumeration is permanent for NICs regardless of mPCIe WiFi module presence or faulty detection.
```
Or it is working differently?

Reply:
It is how it works. I'm changing it to what you suggested.